src/test/java/org/jenkinsci/plugins/docker/commons/credentials/DockerRegistryEndpointTest.java

```diff
@@ -34,22 +34,25 @@
  */
 public class DockerRegistryEndpointTest {
 
+	private static final String indexUrl = "https://index.docker.io/v1/";
```

Comment:
Wrong change. We intentionally keep constants in tests for the better readability